website/docs/r/service_principal_password.html.markdown

```diff
@@ -44,7 +44,11 @@ The following arguments are supported:
 
 * `value` - (Required) The Password for this Service Principal.
 
-* `end_date` - (Required) The End Date which the Password is valid until, formatted as a RFC3339 date string (e.g. `2018-01-01T01:02:03Z`). Changing this field forces a new resource to be created.
+* `end_date` - (Optional) The End Date which the Password is valid until, formatted as a RFC3339 date string (e.g. `2018-01-01T01:02:03Z`). Changing this field forces a new resource to be created.
+
+* `end_date_relative` - (Optional) A relative duration for which the Password is valid until, for example `240h` (10 days) or `2400h30m`. Changing this field forces a new resource to be created.
```

Comment:
I feel like days and hours may be a more realistic example?

Reply:
unfortunately the built in duration parser doesn't support days :(